scripts/update-translation-key.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var fs = require('fs');
```

Comment:
Could we include a comment here that illustrates command line usage?

Reply:
Think we need that in addition to the bit in `DEVELOPMENT.md`?

Reply:
Oh, I just didn't see that.